common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -732,6 +733,7 @@ def associate_by_email_if_login_api(auth_entry, backend, details, user, current_
 
     This association is done ONLY if the user entered the pipeline through a LOGIN API.
     """
+    set_custom_attribute('auth_entry', auth_entry)
     if auth_entry == AUTH_ENTRY_LOGIN_API:
         association_response = associate_by_email(backend, details, user, *args, **kwargs)
```

Comment:
Is this if-block what you want to ensure is not run?  If so, consider adding an additional metric here:
```
        # Temporary custom attribute to help ensure there is no usage.
        set_custom_attribute('deprecated_auth_entry_login_api', True)
```
In addition to this monitoring, did someone follow the DEPR process just in case anyone else still has this configured?

Reply:
@robrap I'm afraid not, an open-source contributor is claiming that and removing this check-in an open PR. I was in doubt and confirming it before we can merge his work.

Reply:
Done.

Reply:
Got it.  I don't imagine it is ever too late for DEPR as a communication mechanism.  Might want to request that on the other PR.  You can ask in Slack if you want more support around this.